qovery/resource_cluster.go

```diff
@@ -154,6 +154,11 @@ func (r clusterResource) Schema(_ context.Context, _ resource.SchemaRequest, res
 					validators.NewStringEnumValidator(clusterKubernetesModes),
 				},
 			},
+			"production": schema.BoolAttribute{
+				Description: "Specific flag to indicate that this cluster is a production one.",
+				Optional:    true,
+				Computed:    true,
```

Comment:
I don't if it is better to not set computed to true but a default value?

Reply:
done